www/resources/views/partials/timings.blade.php

```diff
@@ -0,0 +1,55 @@
+<div class="scrollableTable">
+    <table class="pretty scrollable" cellpadding="10" cellspacing="0" border="1">
+        <thead>
+            <tr>
+                @foreach ($data as $category => $items)
+                @if (!empty($items))
+                <th colspan="{{ count($items) }}">
+                    @switch($category)
+                    @case('custom')
+                    <a href="https://docs.webpagetest.org/custom-metrics/" target="_blank">Custom Metrics</a>
+                    @break
+                    @case('element')
+                    <a href="https://developer.mozilla.org/en-US/docs/Web/API/PerformanceElementTiming" target="_blank">Element Timings</a>
+                    @break
+                    @case('user')
+                    <a href="https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API/Using_the_User_Timing_API" target="_blank">User Timings</a>
+                    @break
+                    @case('navigation')
+                    <a href="https://w3c.github.io/navigation-timing/#processing-model" target="_blank">Navigation Timings</a>
+                    @break
+                    @default
+                    {{ $category }}
+                    @endswitch
+                </th>
+                @endif
+                @endforeach
+            </tr>
+            <tr>
+                @foreach ($data as $category => $items)
+                @foreach ($items as $itemcategory => $_)
+                <th>
+                    @if ($category === 'navigation')
+                    <span title="Time it took for all event handlers to run">{{ $itemcategory }}</span>
+                    @else
+                    {{ $itemcategory }}
+                    @endif
+                </th>
+                @endforeach
```

Comment:
Little thing but does our formatting not pick up on this indent? (I missed while reading because it feels like the indent should nest here)

Reply:
yeah, that irks me too, the VS code formatter doesn't indent these and and puts them on the same column. I end up doing it by hand only to have them undone if I'm not careful, like:
```html
@foreach
@if
<div/>
@endif
@endforeach
```
The php formatter doesn't know about `@`s so ignores them

Reply:
Reformatted the offenders https://github.com/WPO-Foundation/webpagetest/pull/2678/commits/951f8926ec62a27bd2784ac0f2b4a8a34f3e3f68